### PR TITLE
Maven

### LINKED ISSUE #7: Maven/Kotlin Projekt erstellen
Erstellen eines Maven/Kotlin-Projekts mit der MainClass HelloWorld.kt.
Die Ausgabe der Methode soll "Hello, World." ergeben und eine JAR-Datei bilden.

### PR DESCRIPTION
**Pull request by EdenTewelde**
Created a maven project with a HelloWorld class. It builds a self-contained jar.
Closes #7

### CODE REVIEW

File: target/maven-archiver/pom.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```
Comment (nipafx) [inline target/maven-archiver/pom.properties]: Please remove the `target` folder from the PR.

**Review by nipafx — CHANGES_REQUESTED** [review]
Looks great: good, clean POM without any obvious problems. I can't try it out right now, though, so you'll have to wait until later tonight for a thumbs-up.

I was about to point out the whole "Closes #7" stuff but you figured it out. :smiley:

**Comment by EdenTewelde** [comment]
I saw your comment in the other PR, so I changed it here 😄

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+		xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>ep6-TH-Bingen</groupId>
+	<artifactId>project_alpha_omega</artifactId>
+	<version>1.0</version>
```
Comment (nipafx) [inline pom.xml]: During development, you should add `-SNAPSHOT` here. ([Why?](https://stackoverflow.com/a/5901460/2525313))

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+			<version>${kotlin.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.univocity</groupId>
+			<artifactId>univocity-parsers</artifactId>
+			<version>2.4.1</version>
+		</dependency>
+
+		<dependency>
```
Comment (nipafx) [inline pom.xml]: Please remove the JUnit dependency - it is not needed for #7 and should be added in #12.

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+				<groupId>org.jetbrains.kotlin</groupId>
+				<version>${kotlin.version}</version>
+				<executions>
+					<execution>
+						<id>compile</id>
+						<goals>
+							<goal>compile</goal>
+						</goals>
+					</execution>
+					<execution>
```
Comment (nipafx) [inline pom.xml]: This block is only needed for testing.

**Review by nipafx — CHANGES_REQUESTED** [review]
A few changes and this can be merged.

**Comment by Domm2501** [comment]
Seeing that the changes you requested were made, is it now eligible to merge?

**Comment by nipafx** [comment]
Yes.

**Pull request merged**